src/_data/plugins/eleventy-plugin-tag-cloud

```diff
@@ -0,0 +1,5 @@
+{
+  "npm": "eleventy-plugin-tag-cloud",
+  "author": "gingercheww",
```

Comment:
I think you have a typo in your name.

Reply:
Does the profile picture getter still use twitter? I wasn't fast enough to grab `gingerchew` there

Reply:
Then you might have a typo in #1710 - maybe you can check if the grabbed picture here is correct: https://www.11ty.dev/docs/plugins/community/#:~:text=eleventy%2Dplugin%2Dpoison

Reply:
Weird, I swear I had to use the 2 `w` in a previous submission, but maybe I had it backwards. I'll fix that :)

Reply:
Fixed in #2cf2278